### ISSUE TITLE
Unable to open existing or create new 'boards - cannot initialize WebGL

### ISSUE THREAD

**Issue by pencil-test** [wonderunit/storyboarder#2175]
cannot launch Storyboarder v2.0.1 on M1 13" MacBook Pro running Big Sur 11.1.

It loads the splash window listing recent files, but stops with the following error: "Sorry, Storyboarder is not supported on your device because WebGL could not be initialized."

Cannot create a new board or load an existing one.

Deleting prefs file didn't help. 
Rebooting didn't help.

The weird thing is that Storyboarder successfully launched, and I created a board, and I was looking at the interface - all good. I then quit Storyboarder and fetched my iPad (wanted to use Storyboarder with Sidecar) but couldn't launch it again, with or without Sidecar engaged, or even after rebooting and not launching Sidecar.

![Screen Shot 2020-12-20 at 8 34 05 AM](https://user-images.githubusercontent.com/76397198/102715419-46ceea00-42a3-11eb-905a-abc3b43a9457.jpg)

**Comment by audionerd** [comment]
We haven’t tested Storyboarder on M1 yet. The next release of Storyboarder will be based on Electron 11 and should hopefully have better support.

**Comment by pencil-test** [comment]
Great - thanks!
Can't figure out why it worked once and then suddenly stopped.

Whatever happened to Storyboarder on iPad Pro? Because using it on MacOS w/Sidecar or Duet or Astro or whatever on an iPad is just a workaround for an app that is ideally suited for an iPad Pro and Pencil.

**Comment by just4play** [comment]
same as me ... Apple M1 Pro,waitting for reslove...

**Comment by knightRS** [comment]
The exact same happened to me... 

It initially worked when I downloaded Storyboarder from Chrome after enabling WebGL option. Started a storyboard and then logged off for a few hours. Came back to it, the app opens but when I tried to go back into my storyboard or create a new one I got the same message. 

I have the Macbook Air with Big Sur 11.1 8gb and the M1 chip. Hoping for a resolve soon as I'd love to start using Storyboarder again!

**Comment by pencil-test** [comment]
So, it consistently works when I create a new user, meaning it launches, loads a board or creates a new one, saves everything, quits without error. 

But it consistently fails to launch a second time (with the unable to initialize webGL error). 

So, the clunky workaround i think is to 
1- create a new, temp user, 
2- do all your work in Storyboarder, 
3- quit and copy your work to a shared location outside of newly created user,
4- sign into your regular account and delete the temp user. 

I haven’t yet road tested this, but will soon.

**Comment by pencil-test** [comment]
I just had a chance to test out the above method, and while it does indeed work, it's really clunky, mostly because Sidecar requires the same Apple ID on both the host computer user account and the iPad. It works, but every time you create your temp user, you also have to sign into the same apple ID you're using on the iPad.

However, while testing this, I did learn that Storyboarder works great THE FIRST TIME it's launched in a new user - you can open existing or new boards, make changes, save boards, open them again.. Everything is cool until you quit Storyboarder. After that it won't open again, and I cannot figure out what user files or folders need to be deleted to fix that.

**Comment by knightRS** [comment]
Yeah, I can second that. I tried your method and it worked until I quit out from Storyboarder. Tried again this afternoon and again back to the same thing. Creating a new user appears to be a temporary fix. 

Hopefully the new version comes out soon and fixes the issue!

**Comment by dejo095** [comment]
Having same issue on Ubuntu 20.04.1 LTS

**Comment by setpixel** [comment]
Hi everyone! I have a new build for M1. Would some of you be able to test for me?

**Comment by pencil-test** [comment]
Yup!

**Comment by tonykambo** [comment]
Happy to test too

**Comment by pencil-test** [comment]
> Hi everyone! I have a new build for M1. Would some of you be able to test for me?

I feel like an idiot - where can I download the new build?

**Comment by knightRS** [comment]
> Hi everyone! I have a new build for M1. Would some of you be able to test for me?

Likewise, I'm happy to try it out - where can I download it?

**Comment by JohnMatychuk** [comment]
I’m having the exact same issue. I’d love to test out a new build for m1. Where can I download it?

**Comment by audionerd** [comment]
Storyboarder 3 is out now if you want to test and report any issues you find on Apple Silicon. It requires Rosetta 2.
